README.md

```diff
@@ -1,97 +1,50 @@
-# Quick Start
+# Overview
 
-## Prerequisite
-
-In the latest release, Ape requires:
-
-- Linux or macOS
-- Python 3.8 or later
-
-**Windows**:
-
-1.  Install Windows Subsystem Linux
-    [(WSL)](https://docs.microsoft.com/en-us/windows/wsl/install)
-2.  Choose Ubuntu 20.04 OR Any other Linux Distribution with Python
-    3.8 or later
-
-Check your python command by entering:
-
-```bash
-python3 --version
-```
-
-## Installation
-
-There are 3 ways to install ape: `pipx`, `pip`, or `Docker`.
-
-### via `pipx`
-
-Install `pipx` via their [installation instructions](https://pypa.github.io/pipx/)
-
-Then install `ape` via
+**Ape Framework** is an easy-to-use, beginner friendly Web3 development tool. 
+Users can compile, test, and interact with smart contracts all in one command line session.
+With our **modular plugin system**, Ape supports multiple contract languages and chains.
 
-```bash
-pipx install eth-ape
-```
-
-To install Ape and a list of common, recommended plugins at the same time, do:
+Ape is built by [ApeWorX LTD](https://www.apeworx.io/).
 
-```bash
-pipx install eth-ape'[recommended-plugins]'
-```
+Join our [ApeWorX Discord server](https://discord.gg/apeworx) to stay up to date on new release, plugins and tutorials.
 
-### via `pip`
+If you want to just get started, jump down to our Quick Start guide!
 
-**Suggestion**: Create a virtual environment using `virtualenv` or `venv.`
+# Documentation
 
-You may skip creating a virtual environment if you know you don\'t require one for your use case.
+Read our [technical documenation](https://docs.apeworx.io/ape/stable/) to get a deeper understanding of our open source Framework.
 
-- [virtualenv](https://pypi.org/project/virtualenv/)
-- [venv](https://docs.python.org/3/library/venv.html)
+Read our [academic platform](https://academy.apeworx.io/) will help you master Ape Framework with it tutorials and challenges.
 
-**NOTE**: For MacOS users, we advise installing in a virtual environment to avoid interfering with OS-level site packages.
+## Prerequisite
 
-### virtualenv via `homebrew`
+In the latest release, Ape requires:
 
-* (MacOS Option) Install via homebrew [brew](https://formulae.brew.sh/formula/virtualenv)
+- Linux or macOS
+- Python 3.8 or later
+- **Windows**: Install Windows Subsystem Linux [(WSL)](https://docs.microsoft.com/en-us/windows/wsl/install) with Pyhon 3.8 or later
 
-First, create your virtual environment folder:
+Check your python version in a terminal with `python3 --version`
 
-```bash
-python3 -m venv <path/to/new/env>
-```
+## Installation
 
-Then, activate your virtual environment:
+There are three ways to install ape: `pipx`, `pip`, or `Docker`.
 
-```bash
-source <venv_folder>/bin/activate
-```
+##### Considerations for installing:  
```

Comment:
The `#` symbols should only ever increase by 1 at a time. It does not do anything to make it increase more but it throws off the priorities when rendering the markdown.

The header before this has `##` (2 of them).
That means this one can have anywhere from 1-3 `#` symbols.
Based on the context, I am assuming you want `###`

```suggestion
### Considerations for installing:  
```

Reply:
Yes, it was incremental until i removed and reordered some stuff